afb-types/src/chmgr-types.rs

```diff
@@ -66,6 +66,8 @@ pub enum IsoState {
     Iso2,
     Iso3,
     Iec,
+    Eim,
```

Comment:
EIM is appliable to both iso-2 and iso-3, so I would add another enum like e.g. `PaymentMode`  here. What do you think ?

Reply:
Since we are in charging types I was hesitating if we should add a payment option enum as chmgr type. By the way Eim is appliable on iso-3 or Iso-20 you meant ? iso-3 is a lower level protocol and not related to eim as far as I know. In any case I'm ok for a new enum for payment option, so for that either we have to put AfbDataConverter for the new enum and register it, or add the PaymentOption as a variant in ChargingMsg enum. Which one is more suitable you think ?

Reply:
Sorry, I meant EIM is appliable to iso-2 and iso-20

Reply:
> either we have to put AfbDataConverter for the new enum and register it, or add the PaymentOption as a variant in ChargingMsg enum. Which one is more suitable you think ?

No strong opinion about it

Reply:
+ I forgot that the the reason I put eim and pnc in the IsoState enum is that it allows me to have one argument to switch between 3 states (iec, eim, pnc). Because the ManagerHandle in charge has data_set: Mutex::new(ChargingState::default())

Reply:
So If I add a new paymentoption enum, I have to add it in ChargingState variant which doesn't really fit for me in charging state besides power, iso, auth and +payment option

Reply:
I discussed about it with Romain also, I will put a new enum for payment option as you proposed.